addons/sale_project/models/sale_order.py

```diff
@@ -201,7 +201,6 @@ def _timesheet_create_project_prepare_values(self):
         return {
             'name': '%s - %s' % (self.order_id.client_order_ref, self.order_id.name) if self.order_id.client_order_ref else self.order_id.name,
             'analytic_account_id': account.id,
-            'analytic_tag_ids': [Command.set(self.analytic_tag_ids.ids)],
```

Comment:
This would revert https://github.com/odoo/odoo/commit/271c6223cb10230dc8a4d0b257d03f319a1fe790
Besides if we have an order line product creating just the project, the analytic tag would be lost

Reply:
the issue is when we have multiple line set on just creating a project (a single project will be created for all the lines)
in this case project is not tagged but the tag will still be on the invoice from the order_line

Reply:
> a single project will be created for all the lines
if they share the same configuration, actually we "group" the so lines by project template

> project is not tagged but the tag will still be on the invoice from the order_line

if project is not tagged then any task added manually will not have the tag, which is not ideal, especially after https://github.com/odoo/odoo/commit/271c6223cb10230dc8a4d0b257d03f319a1fe790

Reply:
> if they share the same configuration, actually we "group" the so lines by project template

what should be done in that case ? the current behaviour is only setting the tags of the first order line which is wrong if there is multiple lines

create multiple projects if the analytic tags are different ?

Reply:
imho It seems reasonable to keep the proposed changes in `_timesheet_create_task_prepare_values`, in order to add the sale order line tags to the task tags, but we should keep `_timesheet_create_project_prepare_values` as is, to avoid losing project tags

Reply:
unfortunately this won't solve the current issue since all the task still inherits the tags of the project 
the tags of the first task will be applied to all
the best option I think is to keep the tags on the projects but create a different project for tasks with 'project_only' and different tags

Reply:
hi @kpt-odoo can you tells us which is the intended behaviour in this case

Reply:
hi @kpt-odoo can you tell us which is the intended behavior in this case please